### PR TITLE
LiveIntent UserId module: Add support for bidswitch and medianet ids

### PR DESCRIPTION
**Pull request by 3link**
## Type of change
- [x] Other

## Description of change
This pull request adds eid mappings for bidswitch and medianet ids that now can be resolved via the LiveIntent UserId module.

## Other information
- repository of the backing live-connect module: https://github.com/LiveIntent/live-connect

### CODE REVIEW

File: modules/liveIntentIdSystem.js [hunk 1]
```diff
@@ -171,6 +171,14 @@ export const liveIntentIdSubmodule = {
...
       }
 
+      if (value.bidswitch) {
+        result.bidswitch = { 'id': value.bidswitch }
+      }
+
+      if (value.medianet) {
+        result.medianet = { 'id': value.medianet }
+      }
+
```
Comment (3link) [inline modules/liveIntentIdSystem.js:181]: For the reviewer: is there a better approach to expose multiple ids via a single user id module where each of the ids has its own config to create an eid out of it?

File: modules/userId/eids.js [hunk 2]
```diff
@@ -146,6 +146,24 @@ export const USER_IDS_CONFIG = {
     }
   },
 
+  // bidswitchId
+  'bidswitch': {
+    source: 'bidswitch.com',
+    atype: 508,
```
[the remark this answers is not in the document]
Comment (3link) [reply modules/userId/eids.js:152]: Hi Pat, what is your question referring to? The atype or the configuration as a whole? 

The goal of this PR is to add the `eid` config for two identifiers we are going to expose via our user id module for some of our customers - `bidswitch` and `medianet`. The `atype` above is a customer-specific one. 

I am happy to provide more information if you need it.

[the remark this answers is not in the document]
Comment (patmmccann) [reply modules/userId/eids.js:152]: there should be a pr on the document linked specifying what 508 means

**Comment by ChrisHuie** [comment]
@3link can you remove the package-lock change in this pr please 🙏

File: modules/userId/eids.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (3link) [reply modules/userId/eids.js]: @patmmccann Talked to product. Going to use `atype` 3 instead of 508.

**Comment by 3link** [comment]
> can you remove the package-lock change in this pr please 🙏
@ChrisHuie Done.

**Comment by patmmccann** [comment]
@3link could you review issue #9730 and give your comments?

**Comment by 3link** [comment]
> @3link could you review issue #9730 and give your comments?

@patmmccann Done.

**Comment by 3link** [comment]
@patmmccann @ChrisHuie It looks like the tests are failing. It does not look like it is related to changes in this PR. How do you handle these cases - rerun?

**Comment by 3link** [comment]
> @3link thanks for your comments on 9730. Could you point me to a docs pull clarifying this is opt in behavior for module users?

This PR: https://github.com/prebid/prebid.github.io/pull/4048

**Comment by patmmccann** [comment]
okay so it looks like this needs a docs pull to demonstrate how to request these two additional id's? I don't see other issues with merging other than that.

**Comment by 3link** [comment]
> okay so it looks like this needs a docs pull to demonstrate how to request these two additional id's?

@patmmccann Added an example in [eids.md](https://github.com/prebid/Prebid.js/blob/bfb4c92b7dcedf8c4b42794ec2330fc04eb487c4/modules/userId/eids.md) and opened [this doc PR](https://github.com/prebid/prebid.github.io/pull/4477).

**Review by patmmccann — APPROVED** [review]

**Comment by deepthivenkat** [comment]
Here is the document that was shared during the Identity PMC call with suggestions from LiveIntent
https://docs.google.com/document/d/1oOe8cA8NsT4JmEFs095faJpe8QxktfvGGnF9uBsKqzA/edit?usp=sharing

**Pull request merged**